Formula/caf.rb

```diff
@@ -2,8 +2,8 @@ class Caf < Formula
   # Renamed from libccpa
   desc "Implementation of the Actor Model for C++"
   homepage "https://www.actor-framework.org/"
-  url "https://github.com/actor-framework/actor-framework/archive/0.18.1.tar.gz"
-  sha256 "9d9ba0c4bf41ce7a1d7cae9a0efa83816a610b5e84077a02d02cffb8513a5b6d"
+  url "https://github.com/actor-framework/actor-framework/archive/0.18.2.tar.gz"
+  sha256 "cb55a089ac468d5d526a10259ffcafda2d5a9ad0513bfbe64088d39c2c955c64"
```

Comment:
Won't you still get a checksum mismatch here?
```
❯ curl -sSL https://github.com/actor-framework/actor-framework/archive/0.18.2.tar.gz | sha256sum
03837dbef73a13c3e4cac76c009d018f48d1d97b182277230f204984c9542ccc  -
```

Reply:
Yes, you're right. Sorry for the noise.